cloudant_test.go

```diff
@@ -30,7 +31,12 @@ func TestMain(m *testing.M) {
 
 	// Run tests
 	flag.Parse()
-	os.Exit(m.Run())
+
+	testClient.CreateDB(testDBName)
+	result := m.Run()
+	testClient.DeleteDB(testDBName)
```

Comment:
`CreateDB` and `DeleteDB` here are not necessary. As the tests run in sequence, it deletes the existing DB first (https://github.com/IBM-Bluemix/go-cloudant/pull/1/files#diff-3ad1afeee6a291216cf79d4f5e75bef2L42), then create a new one with the same name (https://github.com/IBM-Bluemix/go-cloudant/pull/1/files#diff-3ad1afeee6a291216cf79d4f5e75bef2L48).